src/CineControl.CinemaService.API/Models/DTOs/CinemaMappingExtensions.cs

```diff
@@ -0,0 +1,45 @@
+using CineControl.CinemaService.API.Models;
+using CineControl.CinemaService.API.Models.DTOs.Cinemas;
+using CineControl.CinemaService.API.Models.DTOs.Theaters;
+using CineControl.CinemaService.API.Models.DTOs.Seats;
+
+namespace CineControl.CinemaService.API.Models.DTOs;
+
+public static class CinemaMappingExtensions
+{
+    public static CinemaResponse ToResponse(this Cinema cinema)
```

Comment:
```csharp
public static List<CinemaResponse > ToResponse(
        this List<CinemaResponse > cinemaResponse 
    )
    {
        return cinemaResponse
            .Select(ToResponse)
            .ToList();
    }
```

to powinno załatwić sprawę potworka w controlerze i sprawić że service zwróci po prostu `IEnumerable<CinemaResponse>`